.gitlab-ci.yml

```
@@ -63,29 +64,20 @@ test:integration:
     - ./run.sh
 
 trigger:mender-dist-packages:
-  image: alpine
   stage: trigger
-  before_script:
-    - apk add --no-cache curl
-  script:
-    - curl -v -f -X POST
-      -F token=$MENDER_DIST_PACKAGES_TRIGGER_TOKEN
-      -F ref=master
-      -F variables[MENDER_CONFIGURE_VERSION]=$CI_COMMIT_REF_NAME
-      https://gitlab.com/api/v4/projects/14968223/trigger/pipeline
-  only:
-    - tags
-    - master
+  rules:
+    - if: $CI_COMMIT_TAG
```

What does this condition really mean? Would it work being explicit with something like:
```suggestion
    - if: '$CI_COMMIT_TAG != ""'
```

---

That is taken straight from the [Gitlab docs](https://docs.gitlab.com/ee/ci/yaml/README.html#common-if-clauses-for-rules) and means _if $CI_COMMIT_TAG is truthy_ (so `$CI_COMMIT_TAG != undefined && ($CI_COMMIT_TAG != "" || $CI_COMMIT_TAG > 0 ||  ...)`). For me personally this looks very natural (as it looks like any other comparison of a potentially truthy value in js, ruby, ...), but I'll happily change if you consider this to be more readable.
Depending on the implementation it might be that the comparison to the empty string may not be enough, considering that the var might not be set at all - but that depends on where the rules section is evaluated I guess...

---

Makes sense, we leave it as is. Thanks!